### PR TITLE
ADD: Restart Button

### PR DESCRIPTION
**Pull request by saumyakpandey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://arjunmahishi.com/type-test",
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
```
Comment (arjunmahishi) [inline package.json:7]: Let's try to minimize the number of 3rd party libraries used. Unless we are borrowing a lot of functionality from it.

File: src/App.js [hunk 2]
```diff
@@ -54,21 +61,26 @@ class App extends React.Component {
...
         <GitHubButton href="https://github.com/arjunmahishi/type-test" data-color-scheme="no-preference: light; light: light; dark: dark;" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star arjunmahishi/type-test on GitHub">Star</GitHubButton>
       </div>
       <h1 className="appName">type-test</h1>
-      <div className="App">       
-        <h2 className="speed">{this.state.speed} wpm</h2>
+      
+      <div className="App">
+        <section className="wpmReset">
+           <h2 className="speed">{this.state.speed} wpm</h2>
+        <Button color="secondary" className="restartBtn" onClick={this.restart}>RESTART</Button>
```
[the remark this answers is not in the document]
Comment (arjunmahishi) [reply src/App.js:73]: Like a refresh icon

**Comment by arjunmahishi** [comment]
Merging this.. Will take care of the comments

File: src/App.js [hunk 3]
```diff
@@ -38,6 +39,12 @@ class App extends React.Component {
     this.timerStarted = false
   }
 
+  restart = () =>{
```
[the remark this answers is not in the document]
Comment (saumyakpandey) [reply src/App.js:42]: Yes will rename it better

File: src/App.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (saumyakpandey) [reply src/App.js:73]: Oh yeah.. CSS was tricky, will read more on it and make it smaller.

File: package.json (hunk 1 above)
Comment (saumyakpandey) [reply package.json:7]: Okay got it.

**Pull request merged**